examples/additional_examples/mixed_energy_force/README.md

```diff
@@ -0,0 +1,66 @@
+## Mixed energy and force methods
+Gaussian approximation potentials make use of energies and forces to fit the potential. 
+However, they need not be of the same level of theory (or particularly well converged,
+as noted in the tutorial review[1]). For high-level coupled-cluster potentials where 
+gradients are expensive to calculate the forces may be evaluated at a different level
+of theory to the energies, provided they are within the 'assumed error' (σ<sub>F</sub>)
+to the true forces.
+
+### Force errors
+Generating frames using a DFTB(3ob) MD simulation and evaluating the force components at 
+several levels of theory (*methane_F.py*) suggests that using hybrid DFT or MP2 forces in
+combination with CCSD(T) energies should be sufficient to train a CCSD(T)-quality GAP. The
+average error is roughly the same as the optimised σ<sub>F</sub> hyperparameter (0.1 eV Å<sup>-1</sup>).
+
+<img src="methane_force_comparison_vs_ccsdt.png" width="840">
+
+Similar results are obtained for (molecular) water.
+
+### Mixed energy and force GAPs
+#### 0. Methane
+Training a GAP on CCSD(T) energies and MP2 forces is sufficient to generate a GAP within
+1 kcal mol-1 of the true CC surface. Using active learning at XTB to sample the configuration 
+space, MP2 forces and CC energies highly accurate methane dynamics can be propagated in just
```

Comment:
'energies highly accurate methane dynamics' doesn't read too well. A comma after energies should fix this